.golangci.yml

```diff
@@ -0,0 +1,31 @@
+run:
+  deadline: 10m
+
+output:
+  format: github-actions,colored-line-number
```

Comment:
I suggest removing this `output` section. golangci-lint will switch output format automatically.

It can detect if it is being run as a GitHub action using env vars and it will switch to appropriate logging automatically. As an example, see this [before](https://github.com/jholdstock/dcrvotingweb/actions/runs/5264579054/jobs/9516052155) and [after](https://github.com/jholdstock/dcrvotingweb/actions/runs/5264622564/jobs/9516152749). They both have a properly formatted and highlighted error which looks like this:

![Screenshot from 2023-06-14 09-12-27](https://github.com/decred/dcrvotingweb/assets/6762864/289728b9-e34c-40d7-81c2-6570c10644ce)

However, when you run locally, both output formats are printed which is unnecessary and spammy.

```bash
$ golangci-lint run 
main.go:75:2: printf: log.Printf format %s reads arg #1, but call has 0 args (govet)
        log.Printf("%s Updating vote information")
```

```bash
$ golangci-lint run --out-format github-actions,colored-line-number
::error file=main.go,line=75,col=2::printf: log.Printf format %s reads arg #1, but call has 0 args (govet)
main.go:75:2: printf: log.Printf format %s reads arg #1, but call has 0 args (govet)
        log.Printf("%s Updating vote information")
```